mmocr/utils/ocr.py

```diff
@@ -400,6 +420,99 @@ def __init__(self,
             if hasattr(model, 'module'):
                 model = model.module
 
+    @staticmethod
+    def tesseract_det_inference(imgs):
+        """Inference image(s) with the tesseract detector.
+
+        Args:
+            imgs (list[ndarray]): images to inference.
+
+        Returns:
+            result (dict): Predicted results.
+        """
+        assert is_type_list(imgs, np.ndarray)
+
+        # Very very gross way to find tessdata :(
+        if sys.platform == 'linux':
+            api = PyTessBaseAPI()
+        elif sys.platform == 'win32':
+            try:
+                p = subprocess.Popen(
+                    'where tesseract', stdout=subprocess.PIPE, shell=True)
+                s = p.communicate()[0].decode('utf-8').split('\\')
+                path = s[:-1] + ['tessdata']
+                tessdata_path = '/'.join(path)
+                api = PyTessBaseAPI(path=tessdata_path)
+            except RuntimeError:
+                raise RuntimeError('Please install tesseract first. '
```

Comment:
Hi Garven！
I am not familiar with the use of subprocess. In my case, the `tessdata_path` seems always to be `tessdata`? I prepared a .traindata file and put it in a file named `tessdata`. However, if the `tessdata` file is not at the root path of current folder, it can't be detected.

Reply:
Yes, I don't know why either, the only way I can think of is just assume that the user may have installed tesseract before, then I can detect the `tesseract.exe` path by shell command

Reply:
Otherwise I would prompt user to install tesseract, and better by installer way that it would put the tesseract.exe to `Path` so that I can find it no matter where it is located